app/views/posts/index.html.erb

```diff
@@ -1,19 +1,16 @@
 <div>
-<%  @user = User.find(params[:user_id]) %>
-  <%= render 'share/user', :user => @user %>
-<% @posts = Post.all %>
-<% @users = User.all %>
-<% if !@posts %>
-  <h4 class="card-title">There are no posts from this user yet<h4>
-<% else %>
+  <%= render 'users/user', :user => @user %>
+  <% @users = User.all %>
 <% @posts.each do |p| %>
-
+<% @comments = p.recent_comments %>
 <% if p.author_id === @user.id %>
-  <%= render 'share/post', :post => p %>
-  <div class="text-center">
-  <a href="/users/<%= @user.id %>/posts/<%= @user.id %>"><button type="button" class="btn btn-primary">Pagination</button></a>
-  </div>
+  <%= render 'post', :post => p %>
+  <%= render 'comment', :user => @users, :comment => @comments, :post => p %>
 <% end %>
 <% end %>
-<% end %>
-</div>
\ No newline at end of file
+<div class = "d-flex flex-row mx-auto " style= "width: 320px; justify-content: space-between">
+<%= link_to "See all posts", user_posts_path(@user.id), 
+class: "list-group-item list-group-item-action list-group-item-success", 
+style: "width: 150px;" %>
+</div>
+</div>
```

Comment:
- [ ] Your project is great 🔥. However, one thing is missing, I can't find the button to create a new post, what I did was enter the create new post URL manually `http://localhost:3000/users/5/posts/new`, and I was able to test the post form and actually, it works. But we need to have a button that opens up the page with the form to create a new post, okay. You can add the link/button here on the post index page, it might look something like this`<%= link_to "Create Post", new_user_post_path(@user.id) %>` I actually did add it here and it worked so you will just add some CSS style to make it look like the comment and like button. This will allow users to click on the `Create Post` button/link, and this opens the page for creating a new post. You can also run the command `rails routes` to see the new_user_post_path route and attach it to the create new post button/link as I showed you above. 👍 

Note: The comment and like forms work as expected because I was able to see the button to create a comment and a like. 👍